src/main/java/core/basesyntax/interfaces/FigureBehavior.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+
+public interface FigureBehavior {
+    void draw();
+}
```

Comment:
```suggestion
public interface Drawable{
    void draw();
}
```
if the SAM (single abstract method) is draw(), the more precise name for interface is Drawable. Similarly, create the same interface for calculating area. like AreaCalculator, make your Abstract class implement it (just in class definition, no need to write these methods, figures will provide actual implementation, abstract class doesn't need to implement all abstract methods)